content/reference/apis/_index.md

```diff
@@ -4,6 +4,7 @@ alwaysopen: false
 weight: 1
 ---
 
+- [Configuration Variables](/reference/apis/config_vars)
```

Comment:
I feel like our urls should also remain consistent with our nomenclature of `configuration_variables`

Reply:
Done